BUILD_COMMANDLINE.md

```diff
@@ -54,6 +54,14 @@
 
      We recommend using the latest stable release of Apache Ant™. If your system has an older version of Apache Ant™ installed you might still give it a try though. If the prompt returns an empty string, please refer to the [Apache Ant™ Installation Instructions](https://ant.apache.org/manual/install.html).
 
+  * Is Verovio installed for generating example images locally?
+
+    To build the images with Verovio, you need Python3 to be installed with the `verovio` and `lxml` modules. These can be installed with
+
+    ```shell
+    pip install verovio lxml
```

Comment:
Note that this means that Verovio and LXML need to be installed globally, and not inside a virtual environment. This may cause problems if you have conflicting global setups for Python.

Reply:
Do you want to add this as a note here?

Reply:
Done.